app/logic/Mail/OWA/OWAAccount.ts

```diff
@@ -34,6 +34,13 @@ export class OWAAccount extends MailAccount {
   readonly port: number = 443;
   readonly tls = TLSSocketType.TLS;
   readonly folderMap = new Map<string, OWAFolder>;
+  /*
```

Comment:
```suggestion
  /**
```

JSDoc needs to start with `/**`